ingress/controllers/gce/main.go

```diff
@@ -83,6 +75,13 @@ var (
 		`Optional, if this controller is running in a kubernetes cluster, use the
 		 pod secrets for creating a Kubernetes client.`)
 
+	// TODO: Consolidate this flag and running-in-cluster. People already use
+	// the first one to mean "running in dev", unfortunately.
+	useRealCloud = flags.Bool("use-real-cloud", false,
```

Comment:
```
--running-in-cluster=false --use-real-cloud=false(default): create localhost apiserver client + fake cloud client
--running-in-cluster=false --use-real-cloud=true: create localhost apiserver client + real cloud client
```
the first is for dev, the second is for running this pod on the master (so it needs to use localhost apiserver client but create resources in gce)